File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/domain/ui/dto/step/ForStepDTO.java
package com.mokatest.platform.demos.domain.ui.dto.step;// for循环操作


import com.mokatest.platform.demos.domain.ui.uiEnum.cycle.ForCycleType;
import lombok.Data;

@Data
public class ForStepDTO extends StepBaseDTO {
    private String stepType;
    // 循环方式
    private ForCycleType cycleType;
    // 循环次数
    private Integer cycleTimes;
    // 循环文件
    private String cycleFile;

}

File: frontEnd/src/types/dto/stepDTO/ForStepDTO.ts
// for循环操作
import {StepBaseDTO} from "@/types/dto/base/StepBaseDTO";
import {ConditionalRelationship} from "@/types/enum/condation/ConditionalRelationship";
import {ForCycleType} from "@/types/enum/cycle/ForCycleType";

export class ForStepDTO extends StepBaseDTO {
    stepType: string = 'FOR';
    // 循环方式
    cycleType: ForCycleType | string = 'TIMES';
    // 循环次数
    cycleTimes?: number = 1;
    // 循环文件
    cycleFile?: string = '';

}






